src/compliance-engine-assessor/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+add_compile_options("-Wall;-Wextra;-Wunused;-Werror;-Wformat;-Wformat-security;-Wno-unused-result")
+set (CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_MODULE_PATH};${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+
+if (CMAKE_COMPILER_IS_GNUCC AND CMAKE_CXX_COMPILER_VERSION VERSION_LESS 4.4.7)
+    message(FATAL_ERROR "gcc-4.4.7 or newer is needed")
+endif()
+
+SET(CMAKE_CONFIGURATION_TYPES ${CMAKE_BUILD_TYPE} CACHE STRING "" FORCE)
+
+project(compliance-engine-assessor)
+set(target_name compliance-engine-assessor)
+
+set(SOURCES
+    Main.cpp
+    Mof.cpp
+    BenchmarkFormatter.cpp
+    JsonFormatter.cpp
+    CompactListFormatter.cpp
+    NestedListFormatter.cpp
+    MmiFormatter.cpp
+)
```

Comment:
nit: move closing ) on last line? Here on line 24 (move to line 23), also lines 31 and 39. Right now these 3 instances are not consistent among themselves.